requirements.txt

```diff
@@ -2,3 +2,4 @@ numpy
 periodictable
 scipy
 versioneer
+xraydb
```

Comment:
Looks like xraydb is on nsls2forge. @klauer Do we want to use that?

Reply:
We should really try to get it on conda-forge

Reply:
From my lack of knowledge I don't fully understand what this means for this ticket. Will we have to create a conda-recipe for the `xraydb` or how does it work? Is conda-recipe only taking packages form `conda-forge` unless we specify otherwise?

Reply:
Working on getting xraydb on conda-forge separately - but it may take a while for conda-forge to accept it. Most of it is entirely out of scope for this PR.

Reply:
ohh that's great, thanks a lot @klauer!

Reply:
This moved much more quickly than expected. It should be available on conda-forge very soon, ready for this PR's purposes.